<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administraci√≥n</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="summary_tab">
      <attribute name="title">
       <string>Sumario</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="stores_tab">
      <attribute name="title">
       <string>Tiendas</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="store_create_button">
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="store_update_button">
         <property name="text">
          <string>Editar</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="store_delete_button">
         <property name="text">
          <string>Borrar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QTableWidget" name="store_table_widget"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
